src/views/Stack/Card.tsx

```diff
@@ -636,7 +646,10 @@ export default class Card extends React.Component<Props> {
                         this.gestureUntraversed,
                         I18nManager.isRTL ? MINUS_ONE_NODE : TRUE_NODE
                       ),
-                      this.gestureUntraversed
+                      multiply(
```

Comment:
Instead of multiplying this in each place, maybe it'll be better to create a single node which multiplies it and reuse that? Like we do for `direction`